### PR TITLE
Update various Python requirements.

### PR DESCRIPTION
**Pull request by jmchilton**

### CODE REVIEW

**Comment by jmchilton** [comment]
@galaxybot test this

File: lib/galaxy/dependencies/pinned-requirements.txt [hunk 1]
```diff
@@ -19,7 +19,7 @@ Paste==2.0.2
 PasteDeploy==1.5.2
 docutils==0.12
 wchartype==0.1
-repoze.lru==0.6
+repoze.lru==0.7
```
Comment (nsoranzo) [inline lib/galaxy/dependencies/pinned-requirements.txt:22]: By looking at py27-unit logs on Travis, that this is built from source.

File: lib/galaxy/dependencies/pinned-requirements.txt [hunk 2]
```diff
@@ -37,19 +37,25 @@ galaxy_sequence_utils==1.0.2
...
+pykwalify==1.6.0
 python-dateutil==2.5.3
 docopt==0.6.2
 
 # Cheetah and dependencies
 Cheetah==2.4.4
 Markdown==2.6.3
 
+# Requests and dependencies
+requests==2.18.4
```
Comment (nsoranzo) [inline lib/galaxy/dependencies/pinned-requirements.txt:49]: Missing `certifi==2017.7.27.1` ?

File: lib/galaxy/dependencies/pinned-requirements.txt (hunk 1 above)
Comment (jmchilton) [reply lib/galaxy/dependencies/pinned-requirements.txt:22]: Good catch - I wonder why that is. Works for me when I force it:

```
~/w/galaxy ❯❯❯ wget https://wheels.galaxyproject.org/packages/repoze.lru-0.7-py2-none-any.whl   collection_state ✭ ✱ ◼
--2017-10-27 12:38:09--  https://wheels.galaxyproject.org/packages/repoze.lru-0.7-py2-none-any.whl
Resolving wheels.galaxyproject.org... 54.172.110.33
Connecting to wheels.galaxyproject.org|54.172.110.33|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 10906 (11K) [application/octet-stream]
Saving to: ‘repoze.lru-0.7-py2-none-any.whl’

repoze.lru-0.7-py2-none-any.w 100%[================================================>]  10.65K  --.-KB/s    in 0s

2017-10-27 12:38:10 (146 MB/s) - ‘repoze.lru-0.7-py2-none-any.whl’ saved [10906/10906]

~/w/galaxy ❯❯❯ . .venv/bin/activate                                                             collection_state ✭ ✱ ◼
(.venv) ~/w/galaxy ❯❯❯ pip install repoze.lru-0.7-py2-none-any.whl                              collection_state ✭ ✱ ◼
Processing ./repoze.lru-0.7-py2-none-any.whl
Installing collected packages: repoze.lru
  Found existing installation: repoze.lru 0.6
    Uninstalling repoze.lru-0.6:
      Successfully uninstalled repoze.lru-0.6
Successfully installed repoze.lru-0.7
```

I'm working another update that includes Babel and where I add that missing certifi - I'll see if it happens locally...

**Comment by jmchilton** [comment]
Closing in favor of #4884 - thanks for the caches @nsoranzo.

**Pull request closed without merging**